src/ullyses/tests/test_wrapper.py

```diff
@@ -20,7 +20,7 @@
 V_HK_ORI = {'name': 'v-hk-ori',
             'url': 'https://stsci.box.com/shared/static/dzw9gt283sz6rnekhtq88jxgcl2tq376.gz'}
 
-RELEASE = 'dr6'
+RELEASE = __release__
```

Comment:
For reviewers: should this match the truth file release? There were previously two "release" values in the tests, one set here as `RELEASE = 'dr6'` and the other import from ullyses_utils as `VERSION`. I may have clobbered the difference by blanket-updating them both with the current package version.

Reply:
Good question. This is probably a question for @stscirij

Reply:
The dr6 is the name of a subdirectory in the inflated zip file pulled from Box.  When dr7 products are made, the directory structure of the box zip file will be changed to match.  I am going to have to re-make the Box files anyway, so this change is perfectly OK